flagd/go.mod

```diff
@@ -7,13 +7,16 @@ require (
 	github.com/dimiro1/banner v1.1.0
 	github.com/mattn/go-colorable v0.1.13
 	github.com/open-feature/flagd/core v0.6.5
-	github.com/open-feature/go-sdk-contrib/providers/flagd v0.1.13
-	github.com/open-feature/go-sdk-contrib/tests/flagd v1.2.3
+	github.com/open-feature/go-sdk-contrib/providers/flagd v0.1.16
+	github.com/open-feature/go-sdk-contrib/tests/flagd v1.2.5-0.20230911141323-0f78059f7c4d
```

Comment:
I'm temporarily using a branch commit here, will update once [this](https://github.com/open-feature/go-sdk-contrib/pull/328) is merged.